audformat/core/database.py

```diff
@@ -558,6 +565,21 @@ def join_field(
                 else:
                     self[table_id] = table.copy()
 
+        # copy media files
+        if copy_media and self.root:
+            for other in others:
+                if len(other.files) > 0 and other.root is None:
+                    raise RuntimeError(
+                        f"The database '{other.name}' has no root folder."
+                    )
+                for file in other.files:
```

Comment:
In principle we could use multi-threading here, the only nasty thing is that we have to create all target folders before we start the jobs to avoid the usual threading problem with `mkdir`.

Reply:
Yes, I was thinking about it before, but we also didn't had a `verbose` argument, so I thought I will skip it for now.
Or do you think we should incorporate it directly?

Reply:
No, it's fine to skip it for now.